Issue with Gatsby 3.0

I found some warning issued during the installation over the new Gatsby 3.0 today. The npm installer is issuing compatibility error as below: 
_

> λ npm install --save gatsby-plugin-next-seo react-helmet-async
> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR!
> npm ERR! While resolving: dad-gatsby-cloud-web-site@1.0.0
> npm ERR! Found: gatsby@3.0.1
> npm ERR! node_modules/gatsby
> npm ERR!   gatsby@"^3.0.1" from the root project
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer gatsby@"^2" from gatsby-plugin-next-seo@1.7.0
> npm ERR! node_modules/gatsby-plugin-next-seo
> npm ERR!   gatsby-plugin-next-seo@"*" from the root project
> npm ERR! 
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
> npm ERR!
> npm ERR! See C:\Users\ajaya\AppData\Local\npm-cache\eresolve-report.txt for a full report.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     C:\Users\ajaya\AppData\Local\npm-cache\_logs\2021-03-04T09_31_53_780Z-debug.log

_


Somehow I did force install the package and found out that the following code at below is no longer working. 

`<GatsbySeo
      title='Simple Usage Example'
      description='A short description goes here.'
    />`


Thank you,

--- a/package.json
+++ b/package.json
@@ -170,7 +170,7 @@
     "typescript": "^3.9.5"
   },
   "peerDependencies": {
-    "gatsby": "^2",
+    "gatsby": "^2.0.0 || ^3.0.0",
     "react-helmet-async": "^1"
   },
   "release": {

